sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1242,6 +1388,9 @@ Exceptions:
   property for more information.
 * Throws with error code `invalid` if the type of the command-group is not a
   kernel execution and a `dynamic_parameter` was registered inside `cgf`.
+* Throws with error code `invalid` if the type of the command-group is
```

Comment:
Minor nit, but I'd rephrase this as the type of the command contained in the command group.